beetmoverscript/utils.py

```diff
@@ -15,21 +15,44 @@ def load_json(path):
         return json.load(fh)
 
 
+def infer_template_args(context):
+    props = context.properties
+    tmpl_key_option = "signed" if context.task["payload"]["update_manifest"] is True else "unsigned"
```

Comment:
nice. this could be confusing to outsiders. maybe put a comment how we determine whether this is a signed task or not?